packages/website/docs/studio/loader_nodes/PDFLoaderNode.md

```diff
@@ -0,0 +1,90 @@
+## ** [ Node ] PDF Loader Node **
+
+** A specialized node for loading and processing PDF documents in a data pipeline. **
+
+---
+
+| Reference | Link |
+| --- | --- |
+| Encre Component | [PDF Loader]() |
+| API | [PDF Loader API]() |
+
+### Overview
+
+**
+
+The PDF Loader Node is essential for integrating PDF document processing into your data pipeline. It allows you to extract content from PDF files, either from a URL or a local file, and convert it into a format suitable for further processing. This node is particularly useful for tasks involving document analysis, text extraction, or any workflow that requires processing PDF content.
+
+Prior to using this node, ensure you have a basic understanding of PDF structure and the types of content you expect to extract. Familiarity with blob handling in JavaScript/TypeScript is also beneficial.
+
+```bash
+npm install pdf-parse
+```
+
+**
+
+<Tabs
+  defaultValue="inputs"
+  values={[
+    {label: 'Inputs', value: 'inputs'},
+    {label: 'Outputs', value: 'outputs'},
+    {label: 'Process Context', value: 'context'},
+  ]}
+>
+
+<TabItem value="inputs">
+
+### Inputs
+
+| Port Name | Data Type | Description | Default Value | Notes |
+| --- | --- | --- | --- | --- |
+| pdf | `string` or `blob` | The PDF content to be processed | (required) | Can be either a URL string pointing to a PDF or a Blob containing PDF data |
```

Comment:
For the Data Type, make sure you use the single quote marks for the defined type names. For example, `'string'` or `'blob'`. This is to tell the users these types are not the TypeScript Runtime Types.